algorithms/scripts/SGDClassifier-master.py

```diff
@@ -0,0 +1,309 @@
+
+# coding: utf-8
+
+# # Create a logistic regression model to predict TP53 mutation from gene expression data in TCGA
+
+# In[1]:
+
+import os
+import urllib
+import random
+import warnings
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+from sklearn import preprocessing, grid_search
+from sklearn.linear_model import SGDClassifier
+from sklearn.cross_validation import train_test_split
+from sklearn.metrics import roc_auc_score, roc_curve
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.feature_selection import SelectKBest
+from statsmodels.robust.scale import mad
+
+
+# In[2]:
+
+get_ipython().magic('matplotlib inline')
+plt.style.use('seaborn-notebook')
+
+
+# ## Specify model configuration
+
+# In[3]:
+
+# We're going to be building a 'TP53' classifier 
+GENE = 'TP53'
+
+
+# In[4]:
+
+# Parameter Sweep for Hyperparameters
+n_feature_kept = 500
```

Comment:
I did a git pull today on my laptop, and updated the local machine-learning directory with no problem.  I couldn't find the file SGDClassifier-templater.ipynb mentioned in machine-learning/algorithms/README.md, but found SGDClassifier-master.ipynb 

So I thought to document this...

I edited machine-learning/algorithms/README.md to make this change (I hope that is correct)
then did
    	git add README.md
	git commit -m 's/SGDClassifier-template.ipynb/SGDClassifier-master.ipynb'
When I tried 'git push', I got a 403 error.  
Do I also have to do a pull request up in 'https://github.com/cognoma/machine-learning/' ? 
Sorry to bother you with such trivialities.